scrub-logs.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+##===---------- scrub-logs.py - generate README from Wiki sources ---------===##
```

Comment:
Oh, yeah. I usually use that naming when I want to discourage imports, because it's really just a tool.